content/platforms.md

```diff
@@ -386,13 +386,11 @@ for platforms that use:
 - `useradd`
 - `curl` or `wget`
 
-### Chef Automate Job Runners
+### Chef Backend
 
 #### Commercial Support
 
-Chef Automate Job Runners are supported on the Commercial Support
-platforms for Chef Automate Server listed above as well as on the
-following platforms:
+The following table lists the commercially-supported platforms for Chef Backend, the high-availability solution for Chef Infra Server:
```

Comment:
```suggestion
The following table lists the commercially-supported platforms for Chef Backend and the high-availability solution for Chef Infra Server:
```

Reply:
Chef Backend is "the high availability solution"

Reply:
How about: 
The following table lists the commercially-supported platforms for Chef Backend, which is the high-availability solution for Chef Infra Server:

Reply:
that's fine with me